src/app/shared/actions/actions-dialog/actions-dialog.component.ts

```diff
@@ -83,6 +86,18 @@ export class ActionsDialogComponent {
     }
   }
 
+  autoFocusFirstFieldOnIOS() {
```

Comment:
@sultanmyrza-numbersprotocol This is a workaround, and please add a comment by following the [engineering guide](https://docs.google.com/document/d/12q9oaf5klkJHsQ6mJCDEYXQJoqRevvoC0Ywm8VX-pnM/edit#heading=h.yip9lpmbof9x). Thanks!